scripts/clean-shrinkwrap.js

```diff
@@ -17,13 +17,28 @@
 
 const _ = require('lodash');
 const path = require('path');
-const jsonfile = require('jsonfile');
-const packageJSON = require('../package.json');
-const shrinkwrapIgnore = _.union(packageJSON.shrinkwrapIgnore, _.keys(packageJSON.optionalDependencies));
-const SHRINKWRAP_PATH = path.join(__dirname, '..', 'npm-shrinkwrap.json');
+const pkg = require('../package.json');
```

Comment:
Could we put this back to `const packageJSON` ? It's what all the rest of the code uses...